lib/port_scanner.py

```diff
@@ -15,25 +15,23 @@ def scan_port(host, port):
         sock.close()
 
 
-def scan_thread(host):
-    global queue
-    # TODO reimplement this loop with multiple ranges and threads
-    while True:
-        port_number = queue.get()
-        scan_port(host, port_number)
-        queue.task_done()
+def scan_thread(host, thread_start, thread_end):
```

Comment:
`thread_start` isn't descriptive (threads is not what this variable stores). `port_start` or `start_port` is better. same with `thread_end`

Reply:
maybe `port_thread_start` and `port_thread_end`?

Reply:
`t_port_start` `t_port_end`?

Reply:
deal